src/libsync/creds/oauth.cpp

```diff
@@ -25,6 +25,11 @@
 #include "creds/httpcredentials.h"
 #include <QRandomGenerator>
 
+namespace {
+// TODO: make theming parameter
+QString SCOPE = QStringLiteral("openid offline_access email");
```

Comment:
One should avoid non-trivial static global object.
Use Q_GLOBAL_STATIC,  or return from a function, or just plain char*